src/main/java/com/ethercamp/harmony/jsonrpc/TransactionResultDTO.java

```diff
@@ -50,10 +51,10 @@ public TransactionResultDTO(Block b, int index, Transaction tx) {
         blockNumber = toJsonHex(b.getNumber());
         transactionIndex = toJsonHex(index);
         from= toJsonHex(tx.getSender());
-        to = tx.getReceiveAddress() == null ? null : toJsonHex(tx.getReceiveAddress());
+        to = tx.getReceiveAddress() == null || tx.getReceiveAddress().length == 0 ? null : toJsonHex(tx.getReceiveAddress());
```

Comment:
could you, please, remove this change, I think we could put it to helper method and did it already because it's not the only place in RPC where address could be empty